### DISCUSSION TITLE
Deleted action still shows as required

### DISCUSSION THREAD

**Discussion by mark-stephenson-cto** [community/community#27599] [Actions] [answered]
I recently removed an action file from our repository as it was no longer required.   
The action itself is no longer running but is still showing as "required" in the checks section at the bottom of the PR.

I have tried recreating the PRs that are showing this but the action remains.
I have tried creating a NEW branch and PR and the old action is still included as required.

Can someone point me to how I can remove this as it is preventing non admins from merging PRs.

Thanks

**Comment by JJ** [comment]
Have you tried disabling the workflow?
![Captura de pantalla de 2022-07-29 13-07-21](https://user-images.githubusercontent.com/500/181746714-307ab254-3074-4963-932a-969f949f2ca7.png)
Even if it's deleted, they will still show up as "enabled" in the "actions" menu.

**Reply by mark-stephenson-cto** [reply]
Thanks JJ - Its no longer in the list of workflows as the file has been deleted.  
I _could_ restore the file just to disable it and then delete it but that seems counter intuitive??

**Comment by mark-stephenson-cto** [comment, marked as answer]
This is resolved.  It was listed in the branch protection rules.  I had assumed that the deletion would cascade to anything that was using it.
Perhaps in future deletion of something like this should result in removal of its participation in rules
